tests/test_raw_file_writing.py

```diff
@@ -47,9 +48,13 @@ def test_raw_file_writing(self):
         hdr = hdu.header
 
         # Test some keywords.
-        self.assertAlmostEqual(hdr['RATEL'], opsim_md['fieldRA'])
-        self.assertAlmostEqual(hdr['DECTEL'], opsim_md['fieldDec'])
-        self.assertAlmostEqual(hdr['ROTANGLE'], opsim_md['rotSkyPos'])
+        if hdr.get('TESTTYPE', None) == 'IMSIM':
+            self.assertAlmostEqual(hdr['RATEL'], opsim_md['fieldRA'])
+            self.assertAlmostEqual(hdr['DECTEL'], opsim_md['fieldDec'])
+        else:  # All other cameras, e.g., LsstCam, LsstComCam, etc..
+            self.assertAlmostEqual(hdr['RA'], opsim_md['fieldRA'])
+            self.assertAlmostEqual(hdr['DEC'], opsim_md['fieldDec'])
+        self.assertAlmostEqual(hdr['ROTANGLE'], opsim_md['rotSkyPos'], places=6)
```

Comment:
I had to bump this to `places=5` to pass on my mac (see below).  I think that's probably still good enough?

```
>       self.assertAlmostEqual(hdr['ROTANGLE'], opsim_md['rotSkyPos'], places=6)
E       AssertionError: 146.24369497377404 != 146.24369132422518 within 6 places (3.649548858675189e-06 difference)
```